recipes/pynput-robocorp-fork/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pynput-robocorp-fork" %}
+{% set version = "1.7.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pynput-robocorp-fork-{{ version }}.tar.gz
+  sha256: 45f835a1642ac9f9baa0ed3e0e8cb79ca36eac8e5736f438350bc7b7d57c70bc
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-lint >=0.5
+    - six
+    - sphinx >=1.3.1
+  run:
+    - enum34  # [py==27]
+    # pyobjc-framework versions need to match
+    - pyobjc-framework-quartz ==5.3  # [osx]
+    - pyobjc-framework-cocoa ==5.3  # [osx]
+    - python
+    - python-xlib >=0.17
+    - six
+
+test:
+  # imports:
```

Comment:
Does it not work?

Reply:
Pynput will not import without X11 / other graphical env. And didn't seem worth the hassle to start working yum configurations. Didn't find a way to only skip the test on linux

Reply:
you can skip the tests by adding `#  [not linux]` at the end of the line. But it's fine.

Reply:
Okay good to know!